gnd/src/main/res/values/strings.xml

```diff
@@ -167,4 +167,7 @@
   <string name="no_basemaps_downloaded">No base maps downloaded.</string>
   <string name="please_wait_logging_in">Logging in…</string>
   <string name="build">Build %s</string>
+
+  <string name="terms_conditions">Terms and Conditions</string>
```

Comment:
I believe at Google we usually call these "Terms of Service". Perhaps better to follow precedent here.

Reply:
noted